modules/cas_cache/volume/vol_blk_utils.c

```diff
@@ -446,23 +274,10 @@ int _cas_blk_identify_type(const char *path, uint8_t *type,
 	if (result)
 		return result;
 
-	if (*type == BLOCK_DEVICE_VOLUME) {
```

Comment:
Also `NVME_CONTROLLER` type does not seem to be used anywhere. And it should no longer be considered a valid type.